hicommonwealth/commonwealth
# Topics table + model semantics

## Link to Issue
Closes: #5330

## Description of Changes
- Renamed `Topics.chain_id` to `Topics.community_id` in the database
- Modified the sequelize model and all it's references to match the new community_id
- Updated `chainId` to `communityId` in the client `Topic` model.

## "How We Fixed It"


## Test Plan
- Login as an admin to any community
- Create a new topic (ensure 'appear in sidebar' option is checked)
- Create a thread using the new topic
- Ensure the thread appears under the correct topic in the sidebar
- Re-order the new topic among the other existing topics. 
- Update the topic description.
- Delete the topic.

## Deployment Plan

1. 

## Other Considerations

-
